bower.json

```diff
@@ -19,19 +19,16 @@
     "tests"
   ],
   "dependencies": {
-    "purescript-aff": "^0.11.1",
     "purescript-aff-coroutines": "^0.2.0",
     "purescript-const": "^0.5.0",
-    "purescript-css": "^0.3.0",
-    "purescript-exists": "^0.2.0",
-    "purescript-foreign": "^0.5.0",
+    "purescript-foreign": "^0.7.0",
     "purescript-free": "7b1566995eba2edfbf157e860850c594c9112e49",
-    "purescript-maybe": "^0.3.3",
     "purescript-nullable": "^0.2.0",
-    "purescript-prelude": "^0.1.1",
-    "purescript-profunctor": "^0.3.0",
     "purescript-simple-dom": "^0.1.0",
     "purescript-unsafe-coerce": "^0.1.0",
     "purescript-void": "^0.3.0"
+  },
+  "resolutions": {
+    "purescript-strings": "^0.7.0"
```

Comment:
This resolution is because of a dependency in `simple-dom`, but I'm hoping to replace that with `purescript-dom` before we release this anyway - I've been doing some work on it.

`simple-dom` will be broken by psc 0.7.3 as it has orphan instances, so that's another reason to switch over asap.